wordpress-mobile/WordPress-iOS
# Use readable layout margins for login, sign up, and quick start screens

Related PR: wordpress-mobile/WordPressAuthenticator-iOS#636

These values were previously being calculated based on the view's width. This change aligns the leading and trailing margins to the readable width in the interface builder to make it more consistent.

Additionally changes the login epilogue blog cell to use a vertical stack view to properly center align and word wrap the labels.

### To test (sign up screens):
- Log out if logged in
- Tap on sign up and observe margins of components

### To test (login and quick start):
- Login until you get to the site selection. Observe margins of components.
- Tap on a site to get to the quick start screen. Observe margins of components.

## Screenshots (iPhone)

| Before | After |
|--------|------|
| <img src="https://user-images.githubusercontent.com/2454408/152256341-475d4cce-d2ce-4999-8076-3936973bd97c.png"> | <img src="https://user-images.githubusercontent.com/2454408/152264684-63f75f1b-09fc-4409-98c4-a398e776973f.png"> |

| Before | After |
|--------|------|
| <img src="https://user-images.githubusercontent.com/2454408/152256343-ad0e56c5-c3e8-4f1a-b2de-d72bbd070275.png"> | <img src="https://user-images.githubusercontent.com/2454408/152264686-4cafe788-ee4d-4a7d-a75a-49b6dcc65472.png"> |

## Screenshots (iPad portrait)

| Before | After |
|--------|------|
| <img src="https://user-images.githubusercontent.com/2454408/152264852-3d315465-9930-4c12-8f84-b727bcec2c6d.png"> | <img src="https://user-images.githubusercontent.com/2454408/152264829-0bd0c2e8-ca00-4e30-816f-6661bda04a5b.png"> |

| Before | After |
|--------|------|
| <img src="https://user-images.githubusercontent.com/2454408/152264860-10908d02-3822-42fd-bc59-eed0a4e1c198.png"> | <img src="https://user-images.githubusercontent.com/2454408/152264838-706a08b6-15e4-4556-b3b6-61d6d98c20d7.png"> |

## Screenshots (iPad Landscape)

| Before | After |
|--------|------|
| <img src="https://user-images.githubusercontent.com/2454408/152265145-9bd54f73-3f5c-47f7-8f7e-0ae4447d9e4e.png"> | <img src="https://user-images.githubusercontent.com/2454408/152265117-445d61a9-0891-4954-930e-73205968225b.png"> |

| Before | After |
|--------|------|
| <img src="https://user-images.githubusercontent.com/2454408/152265155-a65c526e-e6ba-4638-b184-2eaf5436c116.png"> | <img src="https://user-images.githubusercontent.com/2454408/152265128-48e0f7a9-4ca8-4861-a337-dc998541c3bc.png"> |

## Regression Notes
1. Potential unintended areas of impact
Layout of sign up, login, and quick start screens.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually verified screens

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
